jobs/v4/src/test/java/JobSearchGetJobTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.samples.JobSearchCreateCompany;
+import com.google.samples.JobSearchCreateJob;
+import com.google.samples.JobSearchCreateTenant;
+import com.google.samples.JobSearchDeleteCompany;
+import com.google.samples.JobSearchDeleteJob;
+import com.google.samples.JobSearchDeleteTenant;
+import com.google.samples.JobSearchGetJob;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class JobSearchGetJobTest {
+  private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static final String TENANT_EXT_ID = "EXTERNAL_TEMP_TENANT_ID";
+  private static final String COMPANY_DISPLAY_NAME = "GOOGLE_DEVREL";
+  private static final String COMPANY_EXT_ID = "COMPANY_EXTERNAL_ID";
+  private static final String POST_UNIQUE_ID =
+      String.format(
+          "TEST_POST_ID_%s",
+          UUID.randomUUID().toString().substring(0, 20)); // Posting ID. Unique per job.
+
+  private String tenantId;
+  private String companyId;
+  private String jobId;
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  @Before
+  public void setUp() throws IOException {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+    // create a tenant for job and company
+    JobSearchCreateTenant.createTenant(PROJECT_ID, TENANT_EXT_ID);
```

Comment:
Does this take a long time to run?
Is it worth pre-creating a lot of these setup steps in a `DO_NOT_DELETE` like company / tenant id that the tests can just access?

Reply:
it is fairly quick 3-8sec average

Reply:
I have decided to create pre-configured tenant and company to reduce the complexity. Once I finish it, I will re-request a review from you.